src/drivers/usb/class/usb_desc.h

```diff
@@ -131,31 +148,20 @@
 
 #ifdef BOOTLOADER
 #define USB_DESC_ISERIALNUM_STR_DESC \
-        (18 + BOOTLOADER_VERSION_SHORT_LEN * 2 + BOOTLOADER_VERSION_APPEND_LEN * 2), /* bLength */\
+        (8 + BOOTLOADER_VERSION_LEN * 2), /* bLength */\
         0x03,  /* bDescriptorType */\
         'b', 0,\
-        'b', 0,\
-        '0', 0,\
-        '2', 0,\
-        '.', 0,\
-        'b', 0,\
         'l', 0,\
         ':', 0,\
-        BOOTLOADER_VERSION_SHORT_W16 \
-        BOOTLOADER_VERSION_APPEND_W16
+        BOOTLOADER_VERSION_W16
 #else
 #define USB_DESC_ISERIALNUM_STR_DESC \
-        (18 + DIGITAL_BITBOX_VERSION_SHORT_LEN * 2), /* bLength */\
+        (8 + DIGITAL_BITBOX_VERSION_LEN * 2), /* bLength */\
         0x03,  /* bDescriptorType */\
-        'b', 0,\
-        'b', 0,\
-        '0', 0,\
-        '2', 0,\
-        '.', 0,\
         'f', 0,\
```

Comment:
why keep this prefix and not make it only a version field?

Reply:
Changed it to only version string.